pyfar/io.py

```diff
@@ -89,12 +89,8 @@ def write_wav(signal, filename, overwrite=True):
 
     # Reshape to 2D
     data = data.reshape(-1, data.shape[-1])
-    warnings.warn("Signal flattened to {data.shape[0]} channels.")
-
-    # Check for .wav file extension
-    if filename.split('.')[-1] != 'wav':
-        warnings.warn("Extending filename by .wav.")
-        filename += '.wav'
```

Comment:
is it possible to support pathlib objects and checking the extension? Otherwise users might pass invalid filenames without the wav ending.

Reply:
Found a solution, .wav suffix is now added withouth warning

Reply:
One line for three with more functionality - nice!